Lib/asyncio/selector_events.py

```diff
@@ -348,6 +348,8 @@ def remove_writer(self, fd):
         The maximum amount of data to be received at once is specified by
         nbytes.
         """
+        if isinstance(sock, ssl.SSLSocket):
+            raise TypeError("Socket cannot be of type SSLSocket instance")
```

Comment:
"of type SSLSocket instance" is strange. This should say either "of type SSLSocket" or "an SSLSocket instance".

(below as well)